### PR TITLE
Fix vhost URL & enable Movefile syntax highlithing

### PR DESCRIPTION
**Pull request by nlemoine**
- Fix vhost URL: site_url ≠ home_url
- Add ".yml" extension to enable syntax highlighting by default (see: https://github.com/welaika/wordmove/issues/46)

### CODE REVIEW

**Comment by nlemoine** [comment]
It would also be nice to have custom paths detection. I may propose another PR for this when I have some spare time.

**Comment by miya0001** [comment]
Ah, I made misunderstand about `site_url()`. Thanks! 😄 

But I am sorry and I don't want to add `.yaml` extension.
Because the documentation in Wordmove is written in `Movefile`, so I want to respect the official documentation.

Coould you remove the `.yaml`?

**Comment by nlemoine** [comment]
> Could you remove the .yaml?

I could but let me try first to prove my point first 😉 

In the first place, Wordmove borrowed this file name from some Ruby world convention: Rakefile, Gemfile, etc. Two problems here: 
1. Unlike Rakefile or Gemfile, Movefile is yaml, not ruby
2. Because Wordmove is just a tiny ruby package for WordPress, Movefile syntax is not recognized by well know editors like Atom or Sublime Text making it harder to read and eventually causing easy mistakes. Sure you can add some custom rules in your editor but adding an extension will work everywhere and require no extra configuration.

> Because the documentation in Wordmove is written in Movefile, so I want to respect the official documentation.

Wordmove handles perfectly fine Movefile with either `yml` or `yaml`, see: https://github.com/welaika/wordmove/pull/47
There's nothing wrong with adding the extension, the docs aren't just up to date.  

Basically, adding the extension does not break any requirements. However, It's ok if want to keep the filename as it is.

**Comment by miya0001** [comment]
I tried that and I see that it works fine. 👍 
Also, I sometimes want to do comment out in Movefile but my editor doesn't work fine. hahaha 😄 
But I want to respect the documentation. There is as `Movefile`.

Thanks

**Comment by nlemoine** [comment]
Ok, I removed the extension. Let me know if you change your mind 😄

**Comment by miya0001** [comment]
Hi @nlemoine,

I released v1.6 and I added a `--output` option.
You can use this command like following.

```
$ wp scaffold movefile --output > Movefile.yml
```

**Comment by nlemoine** [comment]
Thanks a lot @miya0001! 👍 

I'm trying to move the file name convention [directly on the `wordmove` package repo](https://github.com/welaika/wordmove/issues/46). 
Ower has already agreed to mention in the docs that a `Movefile` can also be a `Movefile.yml` or `Movefile.yaml` 😃 

I'll keep your repo updated if anything changes. 

Thanks again for considering this request and providing an alternative way to handle this.

**Comment by nlemoine** [comment]
Hi @miya0001 

After a long time, the change is finally here: https://github.com/welaika/wordmove/pull/424/commits/161489fa443fdcb1f51430c84d26e55e58ef8654

You can now assume `movefile.yaml` as a default file 😄
